### ISSUE TITLE
GoToSlide Event

### ISSUE THREAD

**Issue by emilycritter** [CodyHouse/cd-carousel-changelog#17]
Can a new event be added for navigating to a particular slide (by ID or index)?

My particular use case: the carousel is in a modal; when the modal is opened, it should navigate to a particular slide

The current workaround is triggering a click on a `$('.carousel__nav-item')` element.

Thank you!

**Comment by yankeyhotel** [comment]
I need this too

**Comment by mmp7700** [comment]
I could also use this functionality. Thanks!

**Comment by starchild** [comment]
we also require this functionality - is this on the road map at all please?

**Comment by claudia-romano** [comment]
Yes this is a feature we plan on adding. We are in the process of updating all the CodyHouse components. As soon as we move to the carousel, we'll make sure to include this feature too.
